src/Tags.zig

```diff
@@ -21,15 +21,15 @@ const Entry = struct {
     kind: Kind,
 };
 
-arena: *std.heap.ArenaAllocator,
+allocator: std.mem.Allocator,
```

Comment:
The choice to use an arena allocator directly was intentional, because I omitted `deinit`ing any data structures or freeing any memory. The `arena` was a signal that this was okay to do.

However, using a real allocator _is_ a better design pattern, and freeing memory with an arena allocator is essentially a no-op, so I'm fine with this.

We should also go through and make sure we are calling `deinit()` and `free()` wherever appropriate though.

Reply:
Thanks for reminding, I will check in more details.